sbteclipse-core/src/main/scala/com/typesafe/sbteclipse/util/ScalaVersion.scala

```diff
@@ -0,0 +1,19 @@
+package com.typesafe.sbteclipse.util
+
+import util.control.Exception
+
+case class ScalaVersion(era: Int, major: Int, minor: Int) {
+  def isScala210: Boolean = era == 2 && major == 10
+}
+
+object ScalaVersion {
+  private val versionRegex = """(\d+)\.(\d+)\.(\d+)""".r
```

Comment:
This won't match non-release versions, like RCs or milestones. Maybe that's not a problem for Play, but for other projects this might be useful.